pallets/settlement-zksync/src/lib.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024, The Horizen Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use native::ZKSYNC_PROOF_SIZE as PROOF_SIZE;
+pub use native::ZKSYNC_PUBS_SIZE as PUBS_SIZE;
+pub const FULL_PROOF_SIZE: usize = PROOF_SIZE + PUBS_SIZE;
```

Comment:
I like this and make me think we can also change fflonk side from:
`pub const FULL_PROOF_SIZE: usize = 25 * 32;`
to
`pub const FULL_PROOF_SIZE: usize = PROOF_SIZE + PUBS_SIZE;`

Reply:
A later PR is beter